packages/atomic/src/components/insight/atomic-insight-search-box/atomic-insight-search-box.tsx

```diff
@@ -74,11 +64,10 @@ export class AtomicInsightSearchBox {
 
   public initialize() {
     this.id = randomID('atomic-search-box-');
-    this.querySetActions = loadInsightQuerySetActions(this.bindings.engine);
 
     const searchBoxOptions = {
       id: this.id,
-      numberOfSuggestions: this.numberOfSuggestions,
+      numberOfSuggestions: 0,
```

Comment:
Why are we setting this to 0 here?

Reply:
Because we want the fetching of query suggestions to be done by the suggestion-manager, not the top level search box component itself.

By setting this to 0, the controller won't request double query suggestions; This is done to copy what's inside `atomic-search-box`.